### DISCUSSION TITLE
Licensing

### DISCUSSION THREAD

**Discussion by mlforcada** [N0BOY/FT8CN#78] [General]
This is a great app. I just tried 0.91 patch 1 (APK), just placing next to my radio and pressing the PTT when needed: no cables, just acoustic coupling. It took me 10 minutes to understand how it works, and I have just completed four QSOs (OH6PP, F5MYK, US2UW, ES2DR) in 17 m with OH6PP using 10W and a wire.

However, I am curious about the licence used (MIT: https://github.com/N0BOY/FT8CN/blob/release/LICENSE). If FT8CN was written from scratch following public descriptions of FT8, that would be absolutely fine. But, if the code is based on code in WSJT-X or any of its derivatives, the license should be GPL version 3. 

Could you please clarify?

Thanks

**Comment by bg7yoz** [comment]
hi, @mlforcada 

感谢您的关心，android部分与WSJT-X没有任何关系。解码部分参照了[ft8_lib](https://github.com/kgoba/ft8_lib)编写，是“The MIT License”。

73！

bg7yoz
